src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const successMessage = document.createElement('div');
+const errorMessage = document.createElement('div');
+
+successMessage.classList.add('message');
+successMessage.innerText = 'Promise was resolved!';
+errorMessage.classList.add('message', 'error-message');
+errorMessage.innerText = 'Promise was rejected!';
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', (e) => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Reject'));
```

Comment:
The error message 'Reject' is very generic. It is better to provide a more descriptive message that can help to understand the context of the error.